pkg/deploy/generator/resources.go

```diff
@@ -298,6 +324,175 @@ systemctl enable proxy.service
 	}
 }
 
+func (g *generator) borderVmss() *arm.Resource {
+
+	parts := []string{
+		fmt.Sprintf("base64ToString('%s')", base64.StdEncoding.EncodeToString([]byte("#!/bin/sh\nset -ex\n\n"))),
+	}
+
+	for _, variable := range []string{"borderCert", "borderKey"} {
+		parts = append(parts,
+			fmt.Sprintf("'%s='''", strings.ToUpper(variable)),
+			fmt.Sprintf("parameters('%s')", variable),
+			"'''\n'",
+		)
+	}
+
+	trailer := base64.StdEncoding.EncodeToString([]byte(`yum install -y git
+
+# install golang
+GO_VERSION=1.13.15
+curl -sL https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -o /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+mkdir -p /usr/local/go
+tar -C /usr/local/go -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz --strip-components=1 go
+ln -s /usr/local/go/bin/* /usr/bin/
+
+mkdir -p /root/go/src/github.com/jim-minter/
+cd /root/go/src/github.com/jim-minter
+git clone https://github.com/jim-minter/mitm-proxy.git
+
+cd /root/go/src/github.com/jim-minter/mitm-proxy/
+export HOME=/root
+export GOPATH=/usr/local/go
+/usr/local/go/bin/go build ./mitm-proxy.go
+
+base64 -d <<<"$BORDERCERT" > border-ca-cert.pem
+openssl x509 -in border-ca-cert.pem -outform der -out ca-cert.der
+base64 -d <<<"$BORDERKEY" > border-ca-key.pem
+openssl rsa -inform pem -in border-ca-key.pem -outform der -out ca-key.der
+
+cat <<EOL >./config.yml
```

Comment:
What would be the preferred method to lay down this config file?  Should we keep these settings in a separate variable so that it is easily located or is in the VM setup sufficient?